docs/index.html

```diff
@@ -138,37 +138,61 @@ <h2>Some highlighted content</h2>
 				</div>
 			</div>
 		</section>
-		<section class="highlighted-section au-body au-body--dark au-body--alt">
```

Comment:
@KiriHoy does the `highlighted-section` class do anything? I've noticed there is still a `highlighted-section` class in the `main.scss` file?